mqshudown will produce "Illegal number" when there is no process to shutdown

Environment:
Ubuntu 20.04

![image](https://user-images.githubusercontent.com/10397306/195016218-33ed00c9-7423-4f31-9dba-906cf36c6fe3.png)

<img width="816" alt="image" src="https://user-images.githubusercontent.com/10397306/195016656-0df1fd8e-368a-4171-89ec-d2bef89e515b.png">

As described In https://github.com/koalaman/shellcheck/wiki/SC2242, the `exit 1` is the more canonical way.

--- a/distribution/bin/mqshutdown
+++ b/distribution/bin/mqshutdown
@@ -26,7 +26,7 @@ case $1 in
     pid=`ps ax | grep -i 'org.apache.rocketmq.broker.BrokerStartup' |grep java | grep -v grep | awk '{print $1}'`
     if [ -z "$pid" ] ; then
             echo "No mqbroker running."
-            exit -1;
+            exit 1;
     fi
 
     echo "The mqbroker(${pid}) is running..."
@@ -40,7 +40,7 @@ case $1 in
     pid=`ps ax | grep -i 'org.apache.rocketmq.container.BrokerContainerStartup' |grep java | grep -v grep | awk '{print $1}'`
     if [ -z "$pid" ] ; then
             echo "No broker container running."
-            exit -1;
+            exit 1;
     fi
 
     echo "The broker container(${pid}) is running..."
@@ -54,7 +54,7 @@ case $1 in
     pid=`ps ax | grep -i 'org.apache.rocketmq.namesrv.NamesrvStartup' |grep java | grep -v grep | awk '{print $1}'`
     if [ -z "$pid" ] ; then
             echo "No mqnamesrv running."
-            exit -1;
+            exit 1;
     fi
 
     echo "The mqnamesrv(${pid}) is running..."
@@ -68,7 +68,7 @@ case $1 in
     pid=`ps ax | grep -i 'org.apache.rocketmq.controller.ControllerStartup' |grep java | grep -v grep | awk '{print $1}'`
     if [ -z "$pid" ] ; then
             echo "No mqcontroller running."
-            exit -1;
+            exit 1;
     fi
 
     echo "The mqcontroller(${pid}) is running..."
@@ -82,7 +82,7 @@ case $1 in
     pid=`ps ax | grep -i 'org.apache.rocketmq.proxy.ProxyStartup' |grep java | grep -v grep | awk '{print $1}'`
     if [ -z "$pid" ] ; then
             echo "No mqproxy running."
-            exit -1;
+            exit 1;
     fi
 
     echo "The mqproxy(${pid}) is running..."

